### PR TITLE
Teiid 3647

### PR DESCRIPTION
**Pull request by shawkins**

### CODE REVIEW

File: connectors/file/file-api/src/main/java/org/teiid/file/VirtualFileConnection.java [hunk 1]
```diff
@@ -50,6 +50,10 @@
      */
     boolean remove(String path) throws TranslatorException;
 
+    default boolean areFilesUsableAfterClose() {
```
[the remark this answers is not in the document]
Comment (shawkins) [reply connectors/file/file-api/src/main/java/org/teiid/file/VirtualFileConnection.java:53]: Translators can tell the engine if their lobs are usable after the execution/connection is closed.  We weren't doing that in the file translator, but we should as ftp and hdfs lobs are not usable after close.

File: connectors/file/file-hdfs/src/main/java/org/teiid/hdfs/HdfsConnection.java [hunk 2]
```diff
@@ -95,17 +115,6 @@ protected FileSystem createFileSystem(String fsUri, String resourcePath) throws
...
-            FileStatus fileStatus = fileSystem.getFileStatus(path);
-            if(fileStatus.isDirectory()){
-                return convert(path);
-            }
-            if(fileStatus.isFile()) {
-                return new VirtualFile[] {new HdfsVirtualFile(fileSystem, fileStatus)};
-            }
-        } catch (IOException e) {
-            throw new TranslatorException(e);
-        }
```
[the remark this answers is not in the document]
Comment (shawkins) [reply connectors/file/file-hdfs/src/main/java/org/teiid/hdfs/HdfsConnection.java:108]: Yes it should match the file connection logic now.

**Comment by shawkins** [comment]
> I see we have dropped the dependency on hadoop-commons and hadoop-hdfs. So, we won't need them anymore?

That is correct.  I've rebased the changes and completely removed hadoop-common from the parent pom.

> Why are we able to use the miniDFSCluster in the arquillian test? I could not quite follow the comment about the shaded client

mini cluster and the shaded client jars don't work together in the same classloader, which is noted in the jira issue link.  In an arquillian test a wildfly instance is being launched / managed by the unit test so the actual teiid logic is in a different classloader/process than where we launch the mini cluster.

File: connectors/accumulo/translator-accumulo/src/test/java/org/teiid/translator/accumulo/TestAccumuloMetadataProcessor.java [hunk 3]
```diff
@@ -17,61 +17,80 @@
  */
 package org.teiid.translator.accumulo;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.*;
```
Comment (hola-holo) [inline connectors/accumulo/translator-accumulo/src/test/java/org/teiid/translator/accumulo/TestAccumuloMetadataProcessor.java:20]: our static analysis engine detects that your pull request include change from an explicit import to a wildcard import. According to the best practice (http://www.javadude.com/articles/importondemandisevil.html), import star (import on demand) is a bad practice, wildcard import can sometimes cause name conflicts and ambiguities.

**Pull request merged**